libraries/lib-audio-io/AudioIO.cpp

```
@@ -3010,19 +3034,29 @@ void AudioIoCallback::DoPlaythrough(
+                              outputBuffer, framesPerBuffer,
+                              static_cast<unsigned>(deviceChannels));
    }
 
-   // Copy the results to outputMeterFloats if necessary
+   // Copy the results to outputMeterFloats if necessary.  The meter
+   // buffer is sized to the logical channel count, so extract just the
+   // first numPlaybackChannels samples from each device-strided frame.
    if (outputMeterFloats != outputFloats) {
-      for (unsigned i = 0; i < framesPerBuffer*numPlaybackChannels; ++i) {
-         outputMeterFloats[i] = outputFloats[i];
+      for (unsigned f = 0; f < framesPerBuffer; ++f) {
```

Clang (14.0.6)

[warning] loop variable name 'f' is too short, expected at least 2 characters

(readability-identifier-length)